.github/workflows/test-fork-prs.yml

```
@@ -189,7 +189,7 @@ jobs:
           ref: ${{ needs.setup.outputs.commit_id }}
           repository: ${{ github.repository }}
       - name: 'Dependency Review'
-        uses: actions/dependency-review-action@9129d7d40b8c12c1ed0f60400d00c92d437adcce # v4.1.3 https://github.com/actions/dependency-review-action/commit/9129d7d40b8c12c1ed0f60400d00c92d437adcce
+        uses: actions/dependency-review-action@0c155c5e8556a497adf53f2c18edabf945ed8e70
```

Or should we do something like this?

```suggestion
        uses: actions/dependency-review-action@0c155c5e8556a497adf53f2c18edabf945ed8e70 # https://github.com/actions/dependency-review-action/commit/[HASH]
```

---

If this keeps it automated feel like it doesnt hurt to have, just saves the oncall a little time :)